### ISSUE TITLE
Google BigQuery Node Returns Integer Values as String Instead of Number

### ISSUE THREAD

**Issue by AiratHalitov** [n8n-io/n8n#10503]
### Bug Description

I'm encountering an issue with the Google BigQuery node in n8n. When querying integer fields from BigQuery, the node returns the values as strings instead of numbers. This behavior can cause unexpected results in subsequent nodes that expect numeric data types.

Google BigQuery node version 2.1 (Latest)

Output example:
![image](https://github.com/user-attachments/assets/449d6a19-4b8c-4a32-b39c-0c46455edb40)

DB schema:

![image](https://github.com/user-attachments/assets/a6460c3b-e9a4-42cc-8443-51fdf4b5b307)

```json
[
  {
    "name": "storeId",
    "mode": "NULLABLE",
    "type": "INTEGER",
    "description": "",
    "fields": []
  },
  {
    "name": "GLN",
    "mode": "NULLABLE",
    "type": "INTEGER",
    "description": "",
    "fields": []
  },
  {
    "name": "chainGLN",
    "mode": "NULLABLE",
    "type": "INTEGER",
    "description": "",
    "fields": []
  }
]
```

### To Reproduce

1. Set up a Google BigQuery node in n8n.
2. Query a table that contains integer fields.
3. Inspect the output from the BigQuery node.

### Expected behavior

The integer fields should be returned as numbers.

### Operating System

Alpine Linux v3.20

### n8n Version

1.54.4

### Node.js Version

3.4

### Database

PostgreSQL

### Execution mode

queue

**Comment by Joffcom** [comment]
Hey @AiratHalitov,

We have created an internal ticket to look into this which we will be tracking as "GHC-177"

**Comment by AiratHalitov** [comment]
When I try to cast the values ​​to int type it doesn't help:

```
SELECT 
  CAST(storeId AS INT64) AS storeId,
  CAST(GLN AS INT64) AS GLN,
  CAST(chainGLN AS INT64) AS chainGLN,
FROM `{{ $json.dbname }}`
LIMIT 10;
```
![image](https://github.com/user-attachments/assets/7c0001ab-47a8-4c7f-9376-54b99a729914)

**Comment by AiratHalitov** [comment]
Related: https://community.n8n.io/t/google-bigquery-node-returns-integer-values-as-string-instead-of-number/54825

**Comment by Joffcom** [comment]
Hey @AiratHalitov,

This is likely to be where we don't support int64 fully as Javascript has issues with larger numbers. Instead a string is used to make sure the number is not rounded which would cause more issues.

**Comment by janober** [comment]
Fix got released with [`n8n@1.62.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.62.1)
